Updated documentation for tutorials

lauramarcon opened the pull request:
Modified molecule manipulation, species and molecules


***

## Checklist

* [ ] The changes include an associated explanation of how/why
* [ ] Test pass
* [ ] Documentation has been updated
* [ ] Changelog has been updated

lauramarcon committed:
Update molecules.rst

lauramarcon committed:
Update manipulation.rst

Commit:
Linter update

t-young31 commented on doc/examples/manipulation.rst:
```diff
@@ -72,13 +44,44 @@ hydrogen atom in methane
+  >>>   	frag_names=['Me'],
+  >>>   	name='C2H6')
+  >>>   	ethane = ade.Molecule('C2H6.xyz')
+  >>>   	ethane.n_atoms
```
doesn't look indented quite right

t-young31 commented on doc/examples/manipulation.rst:
```diff
@@ -72,13 +44,44 @@ hydrogen atom in methane
+>>>         atoms_to_del=[2],          #  fresh list EVERY time
+>>>         frag_smiles=[smiles],      #  only SMILES, no frag_names
+>>>         name=out_name
+>>>     )
```
needs a two space indent

lauramarcon committed:
Update manipulation.rst

I think that is what you meant by indentation, let me know if it was something different

lauramarcon commented:
@t-young31 I updated the files, let me know if this is what you meant

t-young31 commented on doc/examples/manipulation.rst:
```diff
@@ -72,13 +44,45 @@ hydrogen atom in methane
+>>>     atoms_to_del=[2],          #  fresh list EVERY time
+>>>     frag_smiles=[smiles],      #  only SMILES, no frag_names
+>>>     name=out_name
+>>>   )
```
```suggestion
  >>> from molfunc import print_combined_molecule
  >>>
  >>> fragments = {
  >>>     "NMe2": "CN([*])C",
  >>>     "NH2":  "N[*]",
  >>>     "OH":   "O[*]",
  >>>     "Me":   "C[*]",
  >>>     "F":    "F[*]",
  >>> }
  >>>
  >>> core_xyz = "CH4.xyz"   # methane
  >>>
  >>> for name, smiles in fragments.items():
  >>>
  >>>     out_name = f"CH3_{name}"
  >>>     print(f"Building {out_name}.xyz")
  >>>
  >>>     print_combined_molecule(
  >>>       core_xyz_filename=core_xyz,
  >>>       atoms_to_del=[2],          #  fresh list EVERY time
  >>>       frag_smiles=[smiles],      #  only SMILES, no frag_names
  >>>       name=out_name
  >>>     )
```

t-young31 commented on doc/examples/manipulation.rst:
```diff
@@ -72,13 +44,45 @@ hydrogen atom in methane
+  >>>   	atoms_to_del=[2],
+  >>>   	frag_names=['Me'],
+  >>>   	name='C2H6'
+  >>> )   
```
```suggestion
  >>> )
```

lauramarcon committed:
Update doc/examples/manipulation.rst

Co-authored-by: Tom Young <39765193+t-young31@users.noreply.github.com>

lauramarcon committed:
Update doc/examples/manipulation.rst

Co-authored-by: Tom Young <39765193+t-young31@users.noreply.github.com>

t-young31 reviewed (approved):
thanks @lauramarcon 🚀
